planner/planner.go

```diff
@@ -1021,6 +1021,11 @@ func (p *MyPlanner) SetupRouter(serverPort string) *http.Server {
 		{
 			places.GET("/:id", p.GetPlaceDetails)
 		}
```

Comment:
is it like a user when they login, they will be directed towards this `page` that will do the announce ?